packages/blockchain-wallet-v4-frontend/src/modals/Settings/UpgradeNow/template.success.tsx

```diff
@@ -0,0 +1,249 @@
+  SuccessCartridge
+} from 'components/Cartridge'
+import { FlyoutWrapper } from 'components/Flyout'
+
+import { TIER_TYPES } from '../TradingLimits/model'
+import { Props as OwnProps, SuccessStateType } from '.'
+
+const Wrapper = styled.div`
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+`
+const Title = styled(Text)`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 7px;
+`
+const IconsContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 100%;
+`
+const HeaderWrapper = styled(FlyoutWrapper)`
+  position: fixed;
+  max-width: 480px;
+  background-color: ${(props) => props.theme.white};
+  z-index: 9999;
+`
+const TierTitle = styled(Text)`
+  color: ${(props) => props.theme.grey900};
+  font-size: 16px;
+  font-weight: 600;
+  line-height: 150%;
+`
+const MainContent = styled.div`
+  margin-top: 222px;
+`
+
+const StatusCartridge = styled.div`
+  display: flex;
+  flex-direction: row;
+  position: absolute;
+  right: 40px;
+  > span {
+    padding: 3px 8px;
+  }
+`
+
+const UpgradeContainer = styled.div<{ second?: boolean }>`
+  border-top: 1px solid ${(props) => props.theme.grey000};
+  padding: 40px;
+  &:hover {
+    background-color: rgba(240, 242, 247, 0.32);
+  }
+  ${(props) =>
+    props.second &&
+    css`
+      border-bottom: 1px solid ${(props) => props.theme.grey000};
+    `}
+`
+const UpgradeSubtitle = styled(Text)`
+  color: ${(props) => props.theme.grey900};
+  font-size: 14px;
+  font-weight: 500;
+  margin: 4px 0 16px 0;
+`
+const UpgradeDescription = styled(Text)`
+  color: ${(props) => props.theme.grey900};
+  font-size: 12px;
+  font-weight: 500;
+  margin: 4px 0 16px 0;
+  > span {
+    color: ${(props) => props.theme.grey600};
+  }
+`
+
+type Props = OwnProps & SuccessStateType
+
+const Template: React.FC<Props> = (props) => {
+  const { identityVerificationActions, interestEDDStatus, sddEligible, userData, userTiers } = props
+
+  if (!Array.isArray(userTiers)) {
+    return null
+  }
+
+  const goldTier = userTiers.find((userTier) => userTier.index === TIER_TYPES.GOLD)
+
+  const userCurrentTier = (path(['tiers', 'current'], userData) as number) ?? 0
+  const sddCheckTier =
+    sddEligible && sddEligible.tier === TIER_TYPES.SILVER_PLUS
+      ? TIER_TYPES.SILVER_PLUS
+      : userCurrentTier
+  const currentTier: number | undefined =
+    userCurrentTier === TIER_TYPES.NONE ? userCurrentTier : sddCheckTier
+  const isUserTierZero = userCurrentTier === TIER_TYPES.NONE
+  const isUserTierSilver =
+    currentTier === TIER_TYPES.SILVER || currentTier === TIER_TYPES.SILVER_PLUS
+
+  const isGoldInreview = goldTier.state === 'under_review' || goldTier.state === 'pending'
+
+  return (
+    <Wrapper>
+      <HeaderWrapper>
+        <IconsContainer>
+          <Link onClick={props.handleClose} data-e2e='upgradeNowCloseButton'>
+            <Image width='26px' name='arrow-left' />
```

Comment:
alt!!!

Reply:
our `Image` do not support alt, we might extend it, or just use name for alt